Utils/AppConstants.swift

```diff
@@ -58,4 +58,12 @@ public struct AppConstants {
     return true
 #endif
     }()
+
+    /// Function returning the What's New page for iOS
+    public static func whatsNewURLForVersion(version: String) -> NSURL? {
+        if let whatsNewURL = NSURL(string: "https://support.mozilla.org/1/mobile/\(version)/%OS%/%LOCALE%/new-ios") {
+            return whatsNewURL
```

Comment:
Just

```
return NSURL(string: "…")
```

— you're testing for an optional and returning the value or `nil`, which is redundant.